README.md

```diff
@@ -8,3 +8,17 @@ A small project which is going to use following tech stack:
 - Google Fonts
 
 Collaborators: @vahan11, and other bootcamp students
+
+_About Hovhannes_
+
+> Hovhannes is one of our smart guys. He has a good heart.  
+> He is married and has a child. All in all he is a great guy,   
+> so I suggest adding his name back to our list. 
+
+P. S. 
+
+```Javascript
```

Comment:
I highly appreciate that you've done great work and used code style in the `readme`. Let's remove it for this PR.